src/applications/static-pages/covid-vaccine-updates-cta/On.jsx

```diff
@@ -35,12 +35,32 @@ function OnState({ copy }) {
         content={
           <>
             <p>{copy.cta}</p>
-            <p>{copy.body}</p>
+
+            {copy.expandedEligibilityContent ? (
+              <ul>
+                <li>
+                  <strong>
+                    {copy.expandedEligibilityContent.veteran.boldedNote}
+                  </strong>
+                  {copy.expandedEligibilityContent.veteran.body}
+                </li>
+                <li>
+                  <strong>
+                    {copy.expandedEligibilityContent.nonVeteran.boldedNote}
+                  </strong>
+                  {copy.expandedEligibilityContent.nonVeteran.body}
+                </li>
+              </ul>
+            ) : (
+              ''
+            )}
+
+            <p> {copy.body ? copy.body : ''}</p>
```

Comment:
Not sure but we _might_ need to wrap the <p> tag in the ternary operator too to prevent some floating whitespace

Reply:
fixed!